irispy/sji.py

```diff
@@ -0,0 +1,188 @@
+        create a list of length 1.
+
+    start: step
+        Temporal axis index to create MapCube from
+
+    stop: step
+        Temporal index to stop MapCube at
+
+    skip: step
+        Temporal index to skip over MapCube at
+
+
+    Returns
+    -------
+    iris_cube: sunpy.map.MapCube
+        A map cube of the SJI sequence
+    """
+
+    if type(filelist) == str:
+        filelist = [filelist]
+    iris_cube = sunpy.map.MapCube()
+    counter = 1
+    setup = sunpy.instr.iris.SJI_to_cube(filelist[0])
+    xcen = setup[0].meta.get('xcen')
+    ycen = setup[0].meta.get('ycen')
+    time = datetime.datetime.strptime(setup[0].meta.get('startobs'),
+                                      '%Y-%m-%d' + 'T' + '%H:%M:%S.%f')
+    print(xcen, ycen, time)
+    for fname in filelist[0:]:
+        newmap = sunpy.instr.iris.SJI_to_cube(fname)
+        for frame in newmap[start:stop:skip]:
+            if frame.mean() < frame.max():
+                cmap = cm.get_cmap(frame._get_cmap_name())
+                vmax = frame.mean()+3.*frame.std()
+                frame.plot_settings['cmap'] = cmap
+                frame.plot_settings['norm'] = colors.LogNorm(1, vmax)
+                #  todo: iris_intscale
+                resultmapcube.maps.append(frame)
+
+        print(str(counter)+' of '+str(len(filelist)))
+        counter += 1
+    #  todo: pointing correction(rot_hpc)
+
+    #  Option to overlay sun coordinates
+    if grid:
+        iris_cube[0].draw_grid(linestyle='--', color='orange')
+
+    return iris_cube
+
+
+def save_to_mp4(mc, outputfile, fps=60):
+    '''
+    Set up ffmpeg writer
+    Configures plot axes
+    Save an animation of the MapCube
+
+    Parameters
+    ----------
+    mc: sunpy.map.MapCube
+        Input mapcube
+    Outputfile: str
+        Path name of output file (.mp4)
+    fps: int
+        Desired frames per seconds (default=60)
+
+    Return
+    -----------
+    NoneType
+        mp4 to outputfile path
+    '''
+
+    plt.rcParams['animation.ffmpeg_path'] = '/usr/local/bin/ffmpeg'
+    writer = animation.FFMpegWriter(fps=fps, metadata=dict(artist='SunPy'),
+                                    bitrate=64000)
+    if not writer.isAvailable():
+        path = input('Set FFMpeg path: (ex. /usr/local/bin/ffmpeg)')
+        plt.rcParams['animation.ffmpeg_path'] = path
+    writer = animation.FFMpegWriter(fps=fps, metadata=dict(artist='SunPy'),
+                                    bitrate=64000)
+    if not writer.isAvailable():
+        print('Cannot find available FFMpeg Writer')
+        return
+    plt.rcParams['axes.facecolor'] = 'black'
+    plt.rcParams['xtick.direction'] = 'out'
+    plt.rcParams['ytick.direction'] = 'out'
+    ax = plt.axes(zorder=0)
+
+    ani = mc.plot(ax)
+    ani.save(outputfile, writer=writer)
+    print('Successfully saved to ' + outputfile)
+
+
+def dustbuster(filelist):
+    """
+       Read SJI fits files and return Inpaint-corrected fits files.
```

Comment:
added: 'Image inpainting involves filling in part of an image or video using information from the surrounding area.'